scripts/shebangConvention.fsx

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env -S dotnet fsi
+
+open System
+open System.IO
+
+#r "nuget: Mono.Unix, 7.1.0-final.1.21458.1"
+#load "../src/FileConventions/Library.fs"
+
+let NotInDir (dirName: string) (fileInfo: FileInfo) =
+    not (fileInfo.FullName.Contains $"%c{Path.DirectorySeparatorChar}%s{dirName}%c{Path.DirectorySeparatorChar}")
+
+let invalidFiles = 
+    Directory.GetFiles(".", "*.fsx", SearchOption.AllDirectories) 
+    |> Seq.map (fun pathStr -> FileInfo pathStr)
+    |> Seq.filter (NotInDir "node_modules")
+    |> Seq.filter (NotInDir ".git")
+    |> Seq.filter (NotInDir "bin")
+    |> Seq.filter (NotInDir "obj")
+    |> Seq.filter (NotInDir "DummyFiles")
+    |> Seq.filter (fun fileInfo -> (not (FileConventions.HasCorrectShebang fileInfo) || 
+                                    not (FileConventions.IsExecutable fileInfo)))
```

Comment:
@realmarv I'd rather you remove the call to FileConventions.IsExecutable from this script, because the script name only mentions shebang anyway (and then you can remove the step and the FIXME)